src/SelectMenu/SelectMenuModal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {COMMON} from '../constants'
+import {modalStyles, modalWrapperStyles} from './SelectMenuStyles'
+import SelectMenuHeader from './SelectMenuHeader'
+import theme from '../theme'
+
+const Modal = styled.div`
+  ${modalStyles}
+`
+
+const ModalWrapper = styled.div`
+  ${modalWrapperStyles}
+  ${COMMON}
+`
+
+const SelectMenuModal = ({children, title, ...rest}) => {
```

Comment:
(and exposing `SelectMenuHeader` as well)

Reply:
(these are also all varying degrees of customization that it's up to y'all to decide which you want to support, just listing options)

Reply:
(but I'd like to see at least one of them implemented; I can easily see a use-case where we want to render Octicons in the header. Not rendering a header at all? That's less likely)

Reply:
(and maybe not even something you want to allow as a Design System)

Reply:
I meant to bring this one up to you - I agree it might be nice to leave the header out of the modal and expose so people can have the option of using it

Reply:
Given the changes we're making to `Dialog`, I would be in favor of making `Header` optional and its own component

Reply:
done!